src/app/app.routes.ts

```diff
@@ -107,8 +110,8 @@ const protectedRoutes: Routes = [
   },
 ].map((route) => ({
   ...route,
-  canActivate: [authGuard],
   resolve: { userState: () => inject(UserService).userProfile$ },
+  canActivate: [authGuard],
```

Comment:
why was this moved below the resolve? 🤣

Reply:
I moved it to see if the order mattered 😂 
It doesn't